example.lua

```
@@ -0,0 +1,22 @@
+        local x = y + 10
+    end
+    local result = a + b
+    return result
+end
+
+local function syntaxError()
+    if true then
+        print("This will cause a syntax error")
+    
+    local missingBracket =
+end
```

Luacheck (1.2.0)

[error] expected expression near 'end'

(E011)